### PR TITLE
Adds more drawbacks/vulnerability to pAIs

### PR DESCRIPTION
**Pull request by Ivniinvi**
## About The Pull Request
Emagging a pAI will now rebind it to you directly, instead of just clearing the current master. The message to a pAI about a master change is now far more visible. After being emagged, the pAI is muted for four seconds and paralyzed for six.
EMPing a pAI will now have stronger effects, in addition to the existing effects. A strong EMP will wipe the pAI. (I wanted to find something more creative than just deth but couldn't think of anything, open to suggestions), and a weak EMP will clear their master, allowing them to be rebound.

## Why It's Good For The Game
pAIs are getting more and more useful, on top of the usefulness of having a second sentient being bound to you. At the moment, having a pAI on you as a murder target drastically increases the difficulty of killing someone, given that the pAI can instantly call for help across multiple channels and PDA messages. This makes them easier to deal with across all antagonists and not just traitors, as pretty much every antagonist has access to an EMP of some form or another.

## Changelog
:cl:
tweak: Emagging a pAI will now be more clear to them, as well as muting and paralyzing them for a short duration.
balance: A strong EMP will now wipe a pAI.
balance: A weak EMP will now clear a pAI's master.
/:cl:

### CODE REVIEW

**Comment by KubeRoot** [comment]
The changelog doesn't mention emagging a PAI will instantly rebind them to you, might want to add that

**Comment by PowerfulBacon** [comment]
Good, it's kind of annoying when someone has a device in their pocket that cant be killed and can instantly say exactly what happens to someone over comms. Ideally they wouldnt be able to talk on comms, just hear it since they are meant to help their master learn rather than do stuff with other people

File: code/modules/mob/living/silicon/pai/pai.dm [hunk 1]
```diff
@@ -301,12 +301,22 @@
...
 	if(!pai)
 		return
-	to_chat(user, "<span class='notice'>You override [pai]'s directive system, clearing its master string and supplied directive.</span>")
-	to_chat(pai, "<span class='danger'>Warning: System override detected, check directive sub-system for any changes.'</span>")
-	log_game("[key_name(user)] emagged [key_name(pai)], wiping their master DNA and supplemental directive.")
-	pai.master = null
-	pai.master_dna = null
+	to_chat(pai, "<span class='userdanger'>Warning: System override detected, check directive sub-system for any changes.'</span>")
+	if(!iscarbon(user))
+		to_chat(usr, "<span class='warning'>You don't have any DNA, or your DNA is incompatible with this device!</span>")
```
Comment (Archanial) [inline code/modules/mob/living/silicon/pai/pai.dm]:
```suggestion
		to_chat(user, "<span class='warning'>You don't have any DNA, or your DNA is incompatible with this device!</span>")
```

File: code/modules/mob/living/silicon/pai/pai_defense.dm [hunk 2]
```diff
@@ -12,6 +12,14 @@
 	if(holoform)
 		fold_in(force = TRUE)
 	//Need more effects that aren't instadeath or permanent law corruption.
+	//With pAIs becoming more and more useful, bigger drawbacks are needed to make them easier to counter
+	switch(severity)
+		if(1) //death
+			qdel(src)
```
Comment (Archanial) [inline code/modules/mob/living/silicon/pai/pai_defense.dm:19]: this is stupid, pai just disappearing lol

**Review by Archanial — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/silicon/pai/pai_defense.dm (hunk 2 above)
Comment (AnCopper) [reply code/modules/mob/living/silicon/pai/pai_defense.dm:19]: > this is stupid, pai just disappearing lol

thanos-infinity

Comment (Ivniinvi) [reply code/modules/mob/living/silicon/pai/pai_defense.dm:19]: Qdels the mob (holoform), leaves the card. I'll put a forced holoform disable in so it doesn't just poof.

**Comment by ZombiesVsPlants** [comment]
wait why are the balance changes listed as features in the changelog lmao?

**Comment by Vexylius** [comment]
Instead of just wiping it, you can make it lose all of the downloaded programs. That'll be actually useful as there's no way to do that iirc

**Comment by Ivniinvi** [comment]
> Instead of just wiping it, you can make it lose all of the downloaded programs. That'll be actually useful as there's no way to do that iirc

That completely misses the point, even with no programs they can go run to sec and tell them you killed their master.

**Pull request closed without merging**